mantidproject/mantid
# Fix Help Information in the Instrument Viewer

**Description of work.**
Directed the instrument viewer to the correct documentation (for workbench, rather than plot) and have it open in the qthelp viewer rather than a web page.

Also updated the `.rst` documentation to reflect the actual functionality present in the GUI. 

**To test:**
1. Delete `doctrees`
2. Build the `docs-qthelp` target. 
3. Build and launch workbench
4. Open the instrument viewer on a workspace
5. Click the ``?`` button
6. The workbench help should appear with the new docs.
7. Check the docs for formatting, spelling, and grammar. 

Fixes #33437

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
